src/expr/src/scalar/func.rs

```diff
@@ -2388,6 +2388,157 @@ pub enum BinaryFunc {
     PowerNumeric,
 }
 
+#[cfg(feature = "test-utils")]
+impl proptest::arbitrary::Arbitrary for BinaryFunc {
+    type Parameters = ();
+
+    type Strategy = Union<BoxedStrategy<Self>>;
+
+    fn arbitrary_with(_: Self::Parameters) -> Self::Strategy {
```

Comment:
Can you leave a comment on
* why we do not use the default implementation of this trait
* the difference between the strategy in this method and the default strategy?

Reply:
I linked the [related `proptest` issue](https://github.com/AltSysrq/proptest/issues/152).